core/app/models/spree/stock_item.rb

```diff
@@ -59,9 +58,6 @@ def reduce_count_on_hand_to_zero
     end
 
     private
-      def verify_count_on_hand?
```

Comment:
I can keep this method if desired, but from my searching of the spree repos this would be dead code with this change. But might be safe to keep it.